SparkPost/libby-react
# UX-493 Persist Source Code Pane

What Changed
- Moves the source code renderer to the UI frame from the Preview frame
- Moves source code state into context
- Refactors the page-bus api a bit to be a bit more readable

How to verify
- Run the example, `cd example` `npm run start`
- Verify Source code show and orientation state is persisted between url changes
